everest

```diff
@@ -394,13 +385,13 @@ MSG
         red "Warning: stale /cygdrive/c/ocamlmgw64; continue? [Yn]"
         prompt_yes true "exit 1"
       fi
-      (cd c:/ && wget -O - $OCAML_TARBALL | tar xjvf -)
-      (cd ~ && wget -O - $OPAM_TARBALL | tar xjvf -)
-      (cd ~/.opam/system/lib && find . -iname '*.cmxs' -exec chmod a+x {} \;)
-      (cd ~/.opam/system/lib &&
-        echo -e "destdir=\"$(cygpath -m $(pwd))\"\npath=\"$(cygpath -m $(pwd))\"" > findlib.conf.new &&
-        tail -n +3 findlib.conf >> findlib.conf.new &&
-        mv findlib.conf.new findlib.conf)
+      # Download and Install OPAM
+      (wget $OPAM_URL)
```

Comment:
the parentheses were only necessary because of the `cd`; parentheses launch a sub-shell and the `cd` does not affect the working directory of the main script; you can get rid of them

Reply:
Fixed.